File: 백준/Bronze/2562. 최댓값/최댓값.py
if __name__ == '__main__':
    max_num = 0
    max_n = 0
    for i in range(9):
        num = int(input())
        if num > max_num:
            max_num = num
            max_n = i + 1
    print(max_num)
    print(max_n)

File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int maxNum = 0;
        int maxNumIndex = 0;

        for (int i = 1; i < 10; i++) {
            int num = Integer.parseInt(br.readLine());
            if (num > maxNum) {
                maxNum = num;
                maxNumIndex = i;
            }
        }

        bw.write(maxNum + "\n");
        bw.write(Integer.toString(maxNumIndex));
        bw.flush();

    }

}